.github/pull_request_template.md

```diff
@@ -5,3 +5,5 @@
 - [ ] This pull request is being made by the same account as the accepted application. 
 - [ ] In case of acceptance, the payment will be transferred to the BTC/ETH payment address provided in the application. 
 - [ ] The delivery is according to the [Guidelines for Milestone Deliverables](https://github.com/w3f/General-Grants-Program/blob/master/grants/milestone-deliverables-guidelines.md). 
+
+Link to the application PR: https://github.com/w3f/Open-Grants-Program/pull/XXX < please fill this in.
```

Comment:
Not sure how to write this without getting the feeling that the extra info might confuse people. I would leave it as is.

```suggestion
Link to the application PR: https://github.com/w3f/Open-Grants-Program/pull/XXX < please fill this in. This is so that GitHub links to your delivery from the application.
```